create-ava-rule.js

```diff
@@ -129,16 +130,29 @@ module.exports = function createAvaRule() {
 		}
 	};
 
-	Object.defineProperty(rule, 'isTestFile', {
-		get: function () {
-			return isTestFile;
-		}
-	});
+	rule.isInTestFile = function () {
+		return isTestFile;
+	};
 
-	Object.defineProperty(rule, 'currentTestNode', {
-		get: function () {
-			return currentTestNode;
-		}
+	rule.isTestNode = function (node) {
+		return currentTestNode === node;
+	};
+
+	rule.isInTestNode = function () {
+		return currentTestNode;
+	};
+
+	rule.if = rest(function (predicates) {
```

Comment:
That's only needed for ES3 backwards compatibility.

https://mathiasbynens.be/notes/javascript-properties